test/cql-pytest/cassandra_tests/validation/operations/alter_test.py

```diff
@@ -222,21 +222,23 @@ def testAlterKeyspaceWithNoOptionThrowsConfigurationException(cql, test_keyspace
 
 # Test {@link ConfigurationException} thrown when altering a keyspace to
 # invalid DC option in replication configuration.
-def testAlterKeyspaceWithNTSOnlyAcceptsConfiguredDataCenterNames(cql, test_keyspace, this_dc):
+def testAlterKeyspaceWithNTSOnlyAcceptsConfiguredDataCenterNames(cql, test_keyspace, this_dc, has_tablets):
```

Comment:
As you can see the test is in the cassandra_tests directory, meaning it is a translated Cassandra unit test. We should try to avoid changing them, especially for such temporary hacks that we'll never remember to undo, and this test will forever miss half of it when tablets are enabled!

So the right fix here is to add a "xfail" referring to #16129. The test will temporarily fail, until 16129 is done, and then will start passing.
If you weren't planning to do #16129 soon I would have told you to also do the whole song-and-dance of using "parametrize" to run this test twice, once with tablets and once without, etc. - but I think that's excessive.

Reply:
Reverted the changes and marked with xfail instead.